### PR TITLE
chore: small UI fixes

### LINKED ISSUE #559: style: increase spacing between vote columns
The space between each column on the votes page should be bigger, now it uses `space-x-1`, it should be `space-x-3`
image

### PR DESCRIPTION
**Pull request by bonustrack**
- Fixes #559 
- Add meta title on edit settings page
- Use "Snapshot" instead of "Snapshot X" as app title
- Add spacing in delegate info message

### CODE REVIEW

File: apps/ui/src/components/SpaceDelegates.vue [hunk 1]
```diff
@@ -134,7 +134,7 @@ watchEffect(() => setTitle(`Delegates - ${props.space.name}`));
           v-if="loaded && (delegates.length === 0 || failed)"
           class="px-4 py-3 flex items-center space-x-2"
         >
-          <IH-exclamation-circle class="inline-block" />
+          <IH-exclamation-circle class="inline-block mr-1" />
```
Comment (wa0x6e) [inline apps/ui/src/components/SpaceDelegates.vue:137]: I would just put the `<template>` below inside a `<span>`, which will auto-enable the spacing set by `space-x-2` in the parent element

File: apps/ui/src/composables/useWalletConnect.ts [hunk 2]
```diff
@@ -26,8 +26,8 @@ async function getConnector() {
       projectId: import.meta.env.VITE_WC_PROJECT_ID
     }),
     metadata: {
-      name: 'Snapshot X',
-      description: 'Snapshot X',
+      name: 'Snapshot',
+      description: 'Where decisions get made',
       url: 'https://snapshotx.org/',
```
Comment (wa0x6e) [inline apps/ui/src/composables/useWalletConnect.ts:31]: We don't use the new url `snapshot.box` here ?

Comment (bonustrack) [reply apps/ui/src/composables/useWalletConnect.ts:31]: We should, good point, URL is was wrong here btw snapshotx.org

**Review by Sekhmet — APPROVED** [review]
tACK

I agree we could update domain to snapshot.box in this PR as well.

File: apps/ui/src/components/SpaceDelegates.vue (hunk 1 above)
Comment (wa0x6e) [reply apps/ui/src/components/SpaceDelegates.vue:137]: Should also add a `shrink-0` to the icon

**Review by wa0x6e — APPROVED** [review]
utAck

**Pull request merged**